package.json

```diff
@@ -72,7 +72,7 @@
     "stats": "node ./stats.js",
     "lint": "standard",
     "snyk-protect": "snyk protect",
-    "prepare": "npm run snyk-protect"
+    "prepare": "npm run snyk-protect && npm run build"
```

Comment:
This is only run when it's used as a git dependency. or before it's published to NPM. If you want to develop locally this will just build the prod version whenever you run `npm install`, but that's reasonable to assume anyway.